config/filter.d/cyrus-imap.conf

```diff
@@ -13,7 +13,7 @@ before = common.conf
 
 _daemon = (?:cyrus/)?(?:imap(d|s)?|pop3(d|s)?)
 
-failregex = ^%(__prefix_line)sbadlogin: \S+ ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
+failregex = ^%(__prefix_line)sbadlogin: (\S+)? ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
```

Comment:
`(\S+)? ?` -\> `(?:\S+ )?`
- correct
- non capturing group

Additionally (because we touch this regexp) I tend to remove `.*\]?$` at end of regexp (greedy catch-all anchored at end makes no sense, also if that contains optional bracket)...

So my regexp would be:
```
failregex = ^%(__prefix_line)sbadlogin: (?:\S+ )?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found):
```
@yarikoptic objections?